ui/app/src/App.tsx

```diff
@@ -54,6 +59,9 @@ function App() {
     return Math.random().toString(36).substring(7);
   };
 
+  const { address } = useAccount();
+  const [streamId, setStreamId] = useState<number>(1);
```

Comment:
Why not store the whole stream?
```suggestion
  const [stream, setStream] = useState<Stream | null>(null);
```